packages/core/src/browser/style/decorators.css

```diff
@@ -0,0 +1,55 @@
+.theia-caption-prefix {
```

Comment:
please check that copyright are in place

Reply:
👍 Do you want me to add the copyright to all other CSS files too? I am not sure which one was done by who... Or can I leave the other files as is?